bootstrap.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/sh
 
-# Build React components
-cd /usr/lib/ckan/ckanext-zarr/ckanext/zarr/react/ && \
-    npm install && \
+# Build fjelltopp file uploader js and css files
+FILE_UPLOADER="$CKAN_VENV/src/ckanext-file-uploader"
+if [ -d "$FILE_UPLOADER" ]; then
```

Comment:
So it's these lines that make the file feel like it is written to be general across all projects. Are they needed?  or will we just copy and paste into projects using ckanext-file-uploader?

Reply:
I think I agree with your thoughts on this - maybe we can brainstorm a better solution.